formats/json-tests/jvmTest/src/kotlinx/serialization/json/MissingFieldExceptionWithPathTest.kt

```diff
@@ -0,0 +1,43 @@
+package kotlinx.serialization.json
+
+import kotlinx.serialization.*
+import kotlinx.serialization.json.Json.Default.decodeFromString
+import org.junit.*
+import org.junit.Test
+import java.io.ByteArrayInputStream
+import java.io.ByteArrayOutputStream
+import java.io.PrintWriter
+import java.io.StringWriter
+import kotlin.test.*
+
+class MissingFieldExceptionWithPathTest {
+
+    @Test // Repro for #2212
+    fun testMfeIsNotReappliedMultipleTimes() {
+        val inputMalformed = """{"title": "...","cast": [{}]"""
+        try {
+            Json.decodeFromString<Movie>(inputMalformed)
+            fail("Unreacheable state")
+        } catch (e: MissingFieldException) {
+            val writer = StringWriter()
+            val pw = PrintWriter(writer)
+            e.printStackTrace(pw)
+            pw.close()
+            val i1 = writer.toString().indexOf("at path")
```

Comment:
We can use `message`, can't we?

Reply:
Not really, because then we'll only check for the most outer exception. I wanted to check exactly the whole exception chain